carol/did.json

```
@@ -24,6 +24,7 @@
   "assertionMethod": ["#z6MkpzW2izkFjNwMBwwvKqmELaQcH8t54QL5xmBdJg9Xh1y4"],
   "service": [{
     "id":"#github",
+    "type": "https://github.com/",
```

As long as you're conforming to DID Core, there is a limited list of [values for `"type"` in `service` objects](https://w3c.github.io/did-spec-registries/#service-types).  `LinkedDomains` doesn't really go with the other values here, but perhaps that doesn't matter so much as conforming to the letters of the rules...
```suggestion
    "type": "LinkedDomains",
```

---

@TallTed, thanks for pointing this out.
Could the type registration expectation be excepted here, for now, considering it is a SHOULD rather than a MUST? Or would this be problematic for implementations?

Additional changes would be needed to use `LinkedDomains` - at least to add the term definition to the JSON-LD context. I think it would imply using [Well Known DID Configuration](https://identity.foundation/.well-known/resources/did-configuration/) - e.g. Carol or some other DID subject claiming control of the `did.actor` domain (Or do the actors get their own subdomains?). I've opened a new issue to track this idea: https://github.com/w3c-ccg/did.actor/issues/12

https://www.w3.org/TR/did-core/#services
"The value of the type property MUST be a string or a set of strings. In order to maximize interoperability, the service type and its associated properties SHOULD be registered in the DID Specification Registries [DID-SPEC-REGISTRIES]."

---

You would need the context that defines `LinkedDomains` added to the did document... when did core purged the primary JSON-LD context, it got removed.

---

also I believe "type" is required... making, here is another example of such a use:

https://github.com/w3c/did-test-suite/blob/main/packages/did-core-test-server/suites/implementations/dereferencer-web-transmute.json#L14

https://github.com/w3c/did-test-suite/blob/main/packages/did-core-test-server/suites/implementations/did-web-transmute.json#L111

---

its not clear to me what the required type value should be..., but whatever it is, its context should define it.... I don't think thats true for either of thee current examples.

---

@OR13 thanks for the links. `RawGitHubUserContent` from `did-web-transmute.json#L111` looks consistent with the use here. But I don't see `@context` for it or otherwise know what the term is supposed to expand to...